packages/teams-js/test/resultValidation.ts

```diff
@@ -1,5 +1,6 @@
 import { MessageRequest } from './utils';
 
+/* eslint-disable-next-line recommend-const-enums/recommend-const-enums */
```

Comment:
Can you say more about why this one needs to be suppressed?

Reply:
See above